enterprise/suppress-user/handler.go

```diff
@@ -30,3 +31,12 @@ func (h *handler) IsSuppressedUser(workspaceID, userID, sourceID string) bool {
 	}
 	return suppressed
 }
+
+func (h *handler) GetCreatedAt(workspaceID, userID, sourceID string) time.Time {
+	h.log.Debugf("GetCreatedAt called for workspace: %s, user %s, source %s", workspaceID, userID, sourceID)
+	createdAt, err := h.r.GetCreatedAt(workspaceID, userID, sourceID)
+	if err != nil && !errors.Is(err, model.ErrRestoring) {
+		h.log.Errorf("GetCreatedAt failed for workspace: %s, user: %s, source: %s: %w", workspaceID, userID, sourceID, err)
+	}
+	return createdAt
+}
```

Comment:
I believe we don't need a new method for that

Reply:
Removed it